sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -1375,6 +1375,14 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
       case other => Seq(other)
     }
 
+    def getLikeQuantifierExps(expressions: java.util.List[ExpressionContext]): Seq[Expression] = {
+      if (expressions.isEmpty) {
+        throw new ParseException("Syntax error: expected something between '(' and ')'.", ctx)
```

Comment:
Teradata's exception:
![image](https://user-images.githubusercontent.com/5399861/74003270-fb725c80-49ac-11ea-853a-2a41cde80b64.png)

Reply:
I think should remove `Syntax error: `, because `ParseException` could replace it.